frontend/src/common/styles/button.css

```diff
@@ -0,0 +1,12 @@
+.button {
+    background-color: #3388c7;
+    border-radius: 0.5rem;
+    color: white;
+    font-weight: bold;
+    width: 10rem;
+    margin: auto;
+}
+
+.button:hover {
+    background-color: #0069d9;
+}
```

Comment:
It cannot be common hover, since you have different colors for different cases, but here all the time blue